src/components/profile/ProfileDataTable.js

```diff
@@ -58,17 +58,25 @@ const ProfileDataTable = ({
   const onPhoneInputFocus = useCallback(() => setLockSubmit(true), [setLockSubmit])
   const onPhoneInputChange = useCallback(value => onChange({ ...profile, mobile: value }), [onChange, profile])
   const onPhoneInputBlur = useCallback(() => {
-    setLockSubmit(false)
-    verifyPhone()
-  }, [setLockSubmit, verifyPhone])
+    const { errors: _errors } = profile.validate()
```

Comment:
focus / blur handlers for phone / email are 100% identical.
so let's use just two handlers - onInputBlur / onInputFocus to avoid duplicated code

Reply:
@serdiukov-o-nordwhale they are not identical https://prnt.sc/sh7el0 they uses different validation props and verify fn